libraries/botframework-config/tests/service.test.js

```diff
@@ -8,6 +8,15 @@ describe("Service Tests", () => {
     it("Luis.getEndpoint() returns correct url for region", async () => {
         let luis = new bf.LuisService({ region: "westus" });
         assert.equal(luis.getEndpoint(), `https://westus.api.cognitive.microsoft.com`);
+
+        luis = new bf.LuisService({ region: "virginia" });
```

Comment:
Comment on this and why it is there? New contributors may not know why virginia is special, and that it is related to government.